app/views/users/edit.html.haml

```diff
@@ -0,0 +1,48 @@
+%h1 Edit Profile
+= form_for @user do |f|
+  .row
+    .input-field.col.s12
+      = f.label :email, :class => "active"
```

Comment:
After ruby `1.9` this hash rocket/fat arrow syntax can be updated as `class: ` instead of `:class =>`